Doctests broken?

What are the requirements for running the documentation tests? I'm getting

```
Preprocessing test suite 'test' for distributors-0.3.0.0...
Building test suite 'test' for distributors-0.3.0.0...
[1 of 9] Compiling Examples.Arithmetic ( test/Examples/Arithmetic.hs, dist/build/test/test-tmp/Examples/Arithmetic.o, dist/build/test/test-tmp/Examples/Arithmetic.dyn_o )
[2 of 9] Compiling Examples.Json    ( test/Examples/Json.hs, dist/build/test/test-tmp/Examples/Json.o, dist/build/test/test-tmp/Examples/Json.dyn_o )
[3 of 9] Compiling Examples.Lambda  ( test/Examples/Lambda.hs, dist/build/test/test-tmp/Examples/Lambda.o, dist/build/test/test-tmp/Examples/Lambda.dyn_o )
[4 of 9] Compiling Examples.LenVec  ( test/Examples/LenVec.hs, dist/build/test/test-tmp/Examples/LenVec.o, dist/build/test/test-tmp/Examples/LenVec.dyn_o )
[5 of 9] Compiling Examples.RegString ( test/Examples/RegString.hs, dist/build/test/test-tmp/Examples/RegString.o, dist/build/test/test-tmp/Examples/RegString.dyn_o )
[6 of 9] Compiling Examples.SExpr   ( test/Examples/SExpr.hs, dist/build/test/test-tmp/Examples/SExpr.o, dist/build/test/test-tmp/Examples/SExpr.dyn_o )
[7 of 9] Compiling Examples.SemVer  ( test/Examples/SemVer.hs, dist/build/test/test-tmp/Examples/SemVer.o, dist/build/test/test-tmp/Examples/SemVer.dyn_o )
[8 of 9] Compiling Main             ( test/Main.hs, dist/build/test/test-tmp/Main.o, dist/build/test/test-tmp/Main.dyn_o )
[9 of 9] Compiling Paths_distributors ( dist/build/test/autogen/Paths_distributors.hs, dist/build/test/test-tmp/Paths_distributors.o, dist/build/test/test-tmp/Paths_distributors.dyn_o )
[10 of 10] Linking dist/build/test/test
buildPhase completed in 1 minutes 7 seconds
Running phase: checkPhase
Running 1 test suites...
Test suite test: RUNNING...
src/Control/Lens/Grammar.hs:37:1: error: [GHC-61948]
    Could not find module ‘Control.Lens.PartialIso’.
    Perhaps you meant Control.Lens.Prism (from lens-5.3.6)
    Use :set -v to see a list of the files searched for.

src/Control/Lens/Grammar.hs:38:1: error: [GHC-87110]
    Could not find module ‘Control.Lens.Grammar.BackusNaur’.
    Use :set -v to see a list of the files searched for.

src/Control/Lens/Grammar.hs:39:1: error: [GHC-87110]
    Could not find module ‘Control.Lens.Grammar.Boole’.
    Use :set -v to see a list of the files searched for.

src/Control/Lens/Grammar.hs:40:1: error: [GHC-87110]
    Could not find module ‘Control.Lens.Grammar.Kleene’.
    Use :set -v to see a list of the files searched for.

src/Control/Lens/Grammar.hs:41:1: error: [GHC-87110]
    Could not find module ‘Control.Lens.Grammar.Token’.
    Use :set -v to see a list of the files searched for.

src/Control/Lens/Grammar.hs:42:1: error: [GHC-87110]
    Could not find module ‘Control.Lens.Grammar.Symbol’.
    Use :set -v to see a list of the files searched for.

src/Control/Lens/Grammar.hs:45:1: error: [GHC-87110]
    Could not find module ‘Data.Profunctor.Distributor’.
    Use :set -v to see a list of the files searched for.

src/Control/Lens/Grammar.hs:46:1: error: [GHC-87110]
    Could not find module ‘Data.Profunctor.Filtrator’.
    Use :set -v to see a list of the files searched for.

src/Control/Lens/Grammar.hs:47:1: error: [GHC-61948]
    Could not find module ‘Data.Profunctor.Monadic’.
    Perhaps you meant
      Data.Profunctor.Monad (from profunctors-5.6.3)
      Data.Profunctor.Yoneda (from profunctors-5.6.3)
      Data.Profunctor.Ran (from profunctors-5.6.3)
    Use :set -v to see a list of the files searched for.

src/Control/Lens/Grammar.hs:48:1: error: [GHC-61948]
    Could not find module ‘Data.Profunctor.Monoidal’.
    Perhaps you meant
      Data.Profunctor.Monad (from profunctors-5.6.3)
      Data.Profunctor.Yoneda (from profunctors-5.6.3)
      Data.Profunctor.Choice (from profunctors-5.6.3)
    Use :set -v to see a list of the files searched for.

src/Control/Lens/Grammar.hs:49:1: error: [GHC-61948]
    Could not find module ‘Data.Profunctor.Grammar’.
    Perhaps you meant
      Data.Profunctor.Ran (from profunctors-5.6.3)
      Data.Profunctor.Monad (from profunctors-5.6.3)
      Data.Profunctor.Cayley (from profunctors-5.6.3)
    Use :set -v to see a list of the files searched for.

src/Control/Lens/Grammar.hs:95: failure in expression `:{
semverGrammar :: RegGrammar Char SemVer
semverGrammar = _SemVer
  >?  numberG
  >*< terminal "." >* numberG
  >*< terminal "." >* numberG
  >*< option [] (terminal "-" >* identifiersG)
  >*< option [] (terminal "+" >* identifiersG)
  where
    numberG = iso show read >~ someP (asIn @Char DecimalNumber)
    identifiersG = several1 (sepBy (terminal ".")) (someP charG)
    charG = asIn LowercaseLetter
      <|> asIn UppercaseLetter
      <|> asIn DecimalNumber
      <|> token '-'
:}'
expected: 
 but got: <interactive>:52:18: error: [GHC-76037]
          ^
              Not in scope: type constructor or class ‘RegGrammar’
          

src/Control/Lens/Grammar.hs:210: failure in expression `:{
arithGrammar :: Grammar Char Arith
arithGrammar = ruleRec "arith" sumG
  where
    sumG arith = rule "sum" $
      chain1 Left _Add (sepBy (terminal "+")) (prodG arith)
    prodG arith = rule "product" $
      chain1 Left _Mul (sepBy (terminal "*")) (factorG arith)
    factorG arith = rule "factor" $
      numberG <|> terminal "(" >* arith *< terminal ")"
    numberG = rule "number" $
      _Num . iso show read >? someP (asIn @Char DecimalNumber)
:}'
expected: 
 but got: <interactive>:93:17: error: [GHC-76037]
          ^
              Not in scope: type constructor or class ‘Grammar’
          

src/Control/Lens/Grammar.hs:252: failure in expression `:{
palindromeG :: CtxGrammar Char String
palindromeG = rule "palindrome" $
  satisfied (\wrd -> reverse wrd == wrd) >?< manyP (anyToken @Char)
:}'
expected: 
 but got: <interactive>:111:16: error: [GHC-76037]
          ^
              Not in scope: type constructor or class ‘CtxGrammar’
          

src/Control/Lens/Grammar.hs:342: failure in expression `let rex = fromString "ab|c" :: RegString'
expected: 
 but got: <interactive>:120:32: error: [GHC-76037]
          ^
              Not in scope: type constructor or class ‘RegString’
          

src/Control/Lens/Grammar.hs:447: failure in expression `let bnf = fromString "{start} = foo|bar" :: RegBnf'
expected: 
 but got: <interactive>:126:45: error: [GHC-76037]
          ^
              Not in scope: type constructor or class ‘RegBnf’
          

src/Control/Lens/Grammar.hs:486: failure in expression `putStringLn (regbnfG regexGrammar)'
expected: {start} = \q{regex}
          {alternate} = \q{sequence}(\|\q{sequence})*
          {atom} = (\\q\{)\q{char}*\}|\q{char}|\q{char-class}|\(\q{regex}\)
          {category} = Ll|Lu|Lt|Lm|Lo|Mn|Mc|Me|Nd|Nl|No|Pc|Pd|Ps|Pe|Pi|Pf|Po|Sm|Sc|Sk|So|Zs|Zl|Zp|Cc|Cf|Cs|Co|Cn
          {category-test} = (\\p\{)\q{category}\}|(\\P\{)(\q{category}(\|\q{category})*)\}
          {char} = [^\(\)\*\+\?\[\\\]\^\{\|\}\P{Cc}]|\\\q{char-escaped}
          {char-any} = \[\^\]
          {char-class} = \q{fail}|\q{char-any}|\q{one-of}|\q{not-one-of}|\q{category-test}
          {char-control} = NUL|SOH|STX|ETX|EOT|ENQ|ACK|BEL|BS|HT|LF|VT|FF|CR|SO|SI|DLE|DC1|DC2|DC3|DC4|NAK|SYN|ETB|CAN|EM|SUB|ESC|FS|GS|RS|US|DEL|PAD|HOP|BPH|NBH|IND|NEL|SSA|ESA|HTS|HTJ|VTS|PLD|PLU|RI|SS2|SS3|DCS|PU1|PU2|STS|CCH|MW|SPA|EPA|SOS|SGCI|SCI|CSI|ST|OSC|PM|APC
          {char-escaped} = [\(\)\*\+\?\[\\\]\^\{\|\}]|\q{char-control}
          {expression} = \q{atom}\?|\q{atom}\*|\q{atom}\+|\q{atom}
          {fail} = \[\]
          {not-one-of} = (\[\^)\q{char}+(\q{category-test}?\])
          {one-of} = \[\q{char}+\]
          {regex} = \q{alternate}
          {sequence} = \q{char}*|\q{expression}*
 but got: <interactive>:132:1: error: [GHC-88464]
          ^
              Variable not in scope: putStringLn :: t1 -> t
              Suggested fix: Perhaps use ‘putStrLn’ (imported from Prelude)
          
          <interactive>:132:14: error: [GHC-88464]
              Variable not in scope: regbnfG :: t0 -> t1
          
          <interactive>:132:22: error: [GHC-88464]
              Variable not in scope: regexGrammar
          

src/Control/Lens/Grammar.hs:671: failure in expression `putStringLn (regbnfG regbnfGrammar)'
expected: {start} = \q{regbnf}
          {alternate} = \q{sequence}(\|\q{sequence})*
          {atom} = (\\q\{)\q{char}*\}|\q{char}|\q{char-class}|\(\q{regex}\)
          {category} = Ll|Lu|Lt|Lm|Lo|Mn|Mc|Me|Nd|Nl|No|Pc|Pd|Ps|Pe|Pi|Pf|Po|Sm|Sc|Sk|So|Zs|Zl|Zp|Cc|Cf|Cs|Co|Cn
          {category-test} = (\\p\{)\q{category}\}|(\\P\{)(\q{category}(\|\q{category})*)\}
          {char} = [^\(\)\*\+\?\[\\\]\^\{\|\}\P{Cc}]|\\\q{char-escaped}
          {char-any} = \[\^\]
          {char-class} = \q{fail}|\q{char-any}|\q{one-of}|\q{not-one-of}|\q{category-test}
          {char-control} = NUL|SOH|STX|ETX|EOT|ENQ|ACK|BEL|BS|HT|LF|VT|FF|CR|SO|SI|DLE|DC1|DC2|DC3|DC4|NAK|SYN|ETB|CAN|EM|SUB|ESC|FS|GS|RS|US|DEL|PAD|HOP|BPH|NBH|IND|NEL|SSA|ESA|HTS|HTJ|VTS|PLD|PLU|RI|SS2|SS3|DCS|PU1|PU2|STS|CCH|MW|SPA|EPA|SOS|SGCI|SCI|CSI|ST|OSC|PM|APC
          {char-escaped} = [\(\)\*\+\?\[\\\]\^\{\|\}]|\q{char-control}
          {expression} = \q{atom}\?|\q{atom}\*|\q{atom}\+|\q{atom}
          {fail} = \[\]
          {not-one-of} = (\[\^)\q{char}+(\q{category-test}?\])
          {one-of} = \[\q{char}+\]
          {regbnf} = (\{start\} = )\q{regex}(\LF\q{rule})*
          {regex} = \q{alternate}
          {rule} = \{\q{char}*(\} = )\q{regex}
          {sequence} = \q{char}*|\q{expression}*
 but got: <interactive>:138:1: error: [GHC-88464]
          ^
              Variable not in scope: putStringLn :: t1 -> t
              Suggested fix: Perhaps use ‘putStrLn’ (imported from Prelude)
          
          <interactive>:138:14: error: [GHC-88464]
              Variable not in scope: regbnfG :: t0 -> t1
          
          <interactive>:138:22: error: [GHC-88464]
              Variable not in scope: regbnfGrammar
          

Examples: 63  Tried: 16  Errors: 0  Failures: 7
Testing module documentation in src/Control/Lens/Grammar.hs
Test suite test: FAIL
Test suite logged to: dist/test/distributors-0.3.0.0-test.log
0 of 1 test suites (0 of 1 test cases) passed.
```

We are using `Setup.hs test` and the following inputs:

```
Configuring distributors-0.3.0.0...
Dependency MemoTrie >=0.6 && <1: using MemoTrie-0.6.11
Dependency adjunctions >=4.4 && <5: using adjunctions-4.4.4
Dependency base >=4.15 && <5: using base-4.20.2.0
Dependency bifunctors >=5.5 && <6: using bifunctors-5.6.3
Dependency bytestring >=0.11 && <1: using bytestring-0.12.2.0
Dependency containers >=0.6 && <1: using containers-0.7
Dependency contravariant >=1.5 && <2: using contravariant-1.5.6
Dependency distributive >=0.6 && <1: using distributive-0.6.3
Dependency lens >=5.0 && <6: using lens-5.3.6
Dependency mtl >=2.2 && <3: using mtl-2.3.1
Dependency profunctors >=5.6 && <6: using profunctors-5.6.3
Dependency tagged >=0.8 && <1: using tagged-0.8.9
Dependency template-haskell >=2.17 && <3: using template-haskell-2.22.0.0
Dependency text >=2 && <3: using text-2.1.3
Dependency th-abstraction >=0.4 && <1: using th-abstraction-0.7.2.0
Dependency vector >=0.12 && <1: using vector-0.13.2.0
Dependency witherable >=0.4 && <1: using witherable-0.5
Dependency MemoTrie >=0.6 && <1: using MemoTrie-0.6.11
Dependency adjunctions >=4.4 && <5: using adjunctions-4.4.4
Dependency base >=4.15 && <5: using base-4.20.2.0
Dependency bifunctors >=5.5 && <6: using bifunctors-5.6.3
Dependency bytestring >=0.11 && <1: using bytestring-0.12.2.0
Dependency containers >=0.6 && <1: using containers-0.7
Dependency contravariant >=1.5 && <2: using contravariant-1.5.6
Dependency distributive >=0.6 && <1: using distributive-0.6.3
Dependency distributors: using distributors-0.3.0.0
Dependency doctest >=0.18 && <1: using doctest-0.24.3
Dependency hspec >=2.7 && <3: using hspec-2.11.16
Dependency lens >=5.0 && <6: using lens-5.3.6
Dependency mtl >=2.2 && <3: using mtl-2.3.1
Dependency profunctors >=5.6 && <6: using profunctors-5.6.3
Dependency tagged >=0.8 && <1: using tagged-0.8.9
Dependency template-haskell >=2.17 && <3: using template-haskell-2.22.0.0
Dependency text >=2 && <3: using text-2.1.3
Dependency th-abstraction >=0.4 && <1: using th-abstraction-0.7.2.0
Dependency vector >=0.12 && <1: using vector-0.13.2.0
Dependency witherable >=0.4 && <1: using witherable-0.5
Source component graph:
    component lib
    component test:test dependency lib
Configured component graph:
    component distributors-0.3.0.0-E5GZBHGi1APGXiHU3nSTgE
        include MemoTrie-0.6.11-Fk66g171UWgIOCpVEpoveJ
        include adjunctions-4.4.4-IkNeuptIx1CDBPRCSY2Zlr
        include base-4.20.2.0-815e
        include bifunctors-5.6.3-JVR3AkjoRc4KGJ5Vy7sniv
        include bytestring-0.12.2.0-a6d5
        include containers-0.7-0069
        include contravariant-1.5.6-7hSaP1u8RjTKTSddvXjMPj
        include distributive-0.6.3-B6bRVuaECsEL512WEfobLI
        include lens-5.3.6-2c7h8ZGgUjXLnIhBDExFHk
        include mtl-2.3.1-e216
        include profunctors-5.6.3-61CyhrRiGB38e3km9Dx9hd
        include tagged-0.8.9-KqQqJnHmV60CspyH6DW7hl
        include template-haskell-2.22.0.0-0ffe
        include text-2.1.3-7c24
        include th-abstraction-0.7.2.0-26upf4VAOFr4UYzfUAil5d
        include vector-0.13.2.0-F6YoK1HFRxjKmjuLWahS3P
        include witherable-0.5-BSuKclwvgCnLbGtQSFKtbo
    component distributors-0.3.0.0-7CvoeBWYEbR4cH8hfTUSZR-test
        include MemoTrie-0.6.11-Fk66g171UWgIOCpVEpoveJ
        include adjunctions-4.4.4-IkNeuptIx1CDBPRCSY2Zlr
        include base-4.20.2.0-815e
        include bifunctors-5.6.3-JVR3AkjoRc4KGJ5Vy7sniv
        include bytestring-0.12.2.0-a6d5
        include containers-0.7-0069
        include contravariant-1.5.6-7hSaP1u8RjTKTSddvXjMPj
        include distributive-0.6.3-B6bRVuaECsEL512WEfobLI
        include distributors-0.3.0.0-E5GZBHGi1APGXiHU3nSTgE
        include doctest-0.24.3-BP9v1wls30S1vPhalxUANv
        include hspec-2.11.16-EXcwcwsDSHnJezrnzLmH7G
        include lens-5.3.6-2c7h8ZGgUjXLnIhBDExFHk
        include mtl-2.3.1-e216
        include profunctors-5.6.3-61CyhrRiGB38e3km9Dx9hd
        include tagged-0.8.9-KqQqJnHmV60CspyH6DW7hl
        include template-haskell-2.22.0.0-0ffe
        include text-2.1.3-7c24
        include th-abstraction-0.7.2.0-26upf4VAOFr4UYzfUAil5d
        include vector-0.13.2.0-F6YoK1HFRxjKmjuLWahS3P
        include witherable-0.5-BSuKclwvgCnLbGtQSFKtbo
Linked component graph:
    unit distributors-0.3.0.0-E5GZBHGi1APGXiHU3nSTgE
        include MemoTrie-0.6.11-Fk66g171UWgIOCpVEpoveJ
        include adjunctions-4.4.4-IkNeuptIx1CDBPRCSY2Zlr
        include base-4.20.2.0-815e
        include bifunctors-5.6.3-JVR3AkjoRc4KGJ5Vy7sniv
        include bytestring-0.12.2.0-a6d5
        include containers-0.7-0069
        include contravariant-1.5.6-7hSaP1u8RjTKTSddvXjMPj
        include distributive-0.6.3-B6bRVuaECsEL512WEfobLI
        include lens-5.3.6-2c7h8ZGgUjXLnIhBDExFHk
        include mtl-2.3.1-e216
        include profunctors-5.6.3-61CyhrRiGB38e3km9Dx9hd
        include tagged-0.8.9-KqQqJnHmV60CspyH6DW7hl
        include template-haskell-2.22.0.0-0ffe
        include text-2.1.3-7c24
        include th-abstraction-0.7.2.0-26upf4VAOFr4UYzfUAil5d
        include vector-0.13.2.0-F6YoK1HFRxjKmjuLWahS3P
        include witherable-0.5-BSuKclwvgCnLbGtQSFKtbo
        Control.Lens.Bifocal=distributors-0.3.0.0-E5GZBHGi1APGXiHU3nSTgE:Control.Lens.Bifocal,Control.Lens.Diopter=distributors-0.3.0.0-E5GZBHGi1APGXiHU3nSTgE:Control.Lens.Diopter,Control.Lens.Grammar=distributors-0.3.0.0-E5GZBHGi1APGXiHU3nSTgE:Control.Lens.Grammar,Control.Lens.Grammar.BackusNaur=distributors-0.3.0.0-E5GZBHGi1APGXiHU3nSTgE:Control.Lens.Grammar.BackusNaur,Control.Lens.Grammar.Boole=distributors-0.3.0.0-E5GZBHGi1APGXiHU3nSTgE:Control.Lens.Grammar.Boole,Control.Lens.Grammar.Kleene=distributors-0.3.0.0-E5GZBHGi1APGXiHU3nSTgE:Control.Lens.Grammar.Kleene,Control.Lens.Grammar.Symbol=distributors-0.3.0.0-E5GZBHGi1APGXiHU3nSTgE:Control.Lens.Grammar.Symbol,Control.Lens.Grammar.Token=distributors-0.3.0.0-E5GZBHGi1APGXiHU3nSTgE:Control.Lens.Grammar.Token,Control.Lens.Grate=distributors-0.3.0.0-E5GZBHGi1APGXiHU3nSTgE:Control.Lens.Grate,Control.Lens.Internal.NestedPrismTH=distributors-0.3.0.0-E5GZBHGi1APGXiHU3nSTgE:Control.Lens.Internal.NestedPrismTH,Control.Lens.Monocle=distributors-0.3.0.0-E5GZBHGi1APGXiHU3nSTgE:Control.Lens.Monocle,Control.Lens.PartialIso=distributors-0.3.0.0-E5GZBHGi1APGXiHU3nSTgE:Control.Lens.PartialIso,Control.Lens.Wither=distributors-0.3.0.0-E5GZBHGi1APGXiHU3nSTgE:Control.Lens.Wither,Data.Profunctor.Distributor=distributors-0.3.0.0-E5GZBHGi1APGXiHU3nSTgE:Data.Profunctor.Distributor,Data.Profunctor.Filtrator=distributors-0.3.0.0-E5GZBHGi1APGXiHU3nSTgE:Data.Profunctor.Filtrator,Data.Profunctor.Grammar=distributors-0.3.0.0-E5GZBHGi1APGXiHU3nSTgE:Data.Profunctor.Grammar,Data.Profunctor.Monadic=distributors-0.3.0.0-E5GZBHGi1APGXiHU3nSTgE:Data.Profunctor.Monadic,Data.Profunctor.Monoidal=distributors-0.3.0.0-E5GZBHGi1APGXiHU3nSTgE:Data.Profunctor.Monoidal
    unit distributors-0.3.0.0-7CvoeBWYEbR4cH8hfTUSZR-test
        include MemoTrie-0.6.11-Fk66g171UWgIOCpVEpoveJ
        include adjunctions-4.4.4-IkNeuptIx1CDBPRCSY2Zlr
        include base-4.20.2.0-815e
        include bifunctors-5.6.3-JVR3AkjoRc4KGJ5Vy7sniv
        include bytestring-0.12.2.0-a6d5
        include containers-0.7-0069
        include contravariant-1.5.6-7hSaP1u8RjTKTSddvXjMPj
        include distributive-0.6.3-B6bRVuaECsEL512WEfobLI
        include distributors-0.3.0.0-E5GZBHGi1APGXiHU3nSTgE
        include doctest-0.24.3-BP9v1wls30S1vPhalxUANv
        include hspec-2.11.16-EXcwcwsDSHnJezrnzLmH7G
        include lens-5.3.6-2c7h8ZGgUjXLnIhBDExFHk
        include mtl-2.3.1-e216
        include profunctors-5.6.3-61CyhrRiGB38e3km9Dx9hd
        include tagged-0.8.9-KqQqJnHmV60CspyH6DW7hl
        include template-haskell-2.22.0.0-0ffe
        include text-2.1.3-7c24
        include th-abstraction-0.7.2.0-26upf4VAOFr4UYzfUAil5d
        include vector-0.13.2.0-F6YoK1HFRxjKmjuLWahS3P
        include witherable-0.5-BSuKclwvgCnLbGtQSFKtbo
Ready component graph:
    definite distributors-0.3.0.0-E5GZBHGi1APGXiHU3nSTgE
        depends MemoTrie-0.6.11-Fk66g171UWgIOCpVEpoveJ
        depends adjunctions-4.4.4-IkNeuptIx1CDBPRCSY2Zlr
        depends base-4.20.2.0-815e
        depends bifunctors-5.6.3-JVR3AkjoRc4KGJ5Vy7sniv
        depends bytestring-0.12.2.0-a6d5
        depends containers-0.7-0069
        depends contravariant-1.5.6-7hSaP1u8RjTKTSddvXjMPj
        depends distributive-0.6.3-B6bRVuaECsEL512WEfobLI
        depends lens-5.3.6-2c7h8ZGgUjXLnIhBDExFHk
        depends mtl-2.3.1-e216
        depends profunctors-5.6.3-61CyhrRiGB38e3km9Dx9hd
        depends tagged-0.8.9-KqQqJnHmV60CspyH6DW7hl
        depends template-haskell-2.22.0.0-0ffe
        depends text-2.1.3-7c24
        depends th-abstraction-0.7.2.0-26upf4VAOFr4UYzfUAil5d
        depends vector-0.13.2.0-F6YoK1HFRxjKmjuLWahS3P
        depends witherable-0.5-BSuKclwvgCnLbGtQSFKtbo
    definite distributors-0.3.0.0-7CvoeBWYEbR4cH8hfTUSZR-test
        depends MemoTrie-0.6.11-Fk66g171UWgIOCpVEpoveJ
        depends adjunctions-4.4.4-IkNeuptIx1CDBPRCSY2Zlr
        depends base-4.20.2.0-815e
        depends bifunctors-5.6.3-JVR3AkjoRc4KGJ5Vy7sniv
        depends bytestring-0.12.2.0-a6d5
        depends containers-0.7-0069
        depends contravariant-1.5.6-7hSaP1u8RjTKTSddvXjMPj
        depends distributive-0.6.3-B6bRVuaECsEL512WEfobLI
        depends distributors-0.3.0.0-E5GZBHGi1APGXiHU3nSTgE
        depends doctest-0.24.3-BP9v1wls30S1vPhalxUANv
        depends hspec-2.11.16-EXcwcwsDSHnJezrnzLmH7G
        depends lens-5.3.6-2c7h8ZGgUjXLnIhBDExFHk
        depends mtl-2.3.1-e216
        depends profunctors-5.6.3-61CyhrRiGB38e3km9Dx9hd
        depends tagged-0.8.9-KqQqJnHmV60CspyH6DW7hl
        depends template-haskell-2.22.0.0-0ffe
        depends text-2.1.3-7c24
        depends th-abstraction-0.7.2.0-26upf4VAOFr4UYzfUAil5d
        depends vector-0.13.2.0-F6YoK1HFRxjKmjuLWahS3P
        depends witherable-0.5-BSuKclwvgCnLbGtQSFKtbo
Using Cabal-3.12.1.0 compiled by ghc-9.10
Using compiler: ghc-9.10.3
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -32,4 +32,6 @@ jobs:
         run: stack build
 
       - name: Run tests
+        env:
+          DISTRIBUTORS_RUN_DOCTESTS: "1"
         run: stack test

--- a/test/Main.hs
+++ b/test/Main.hs
@@ -27,7 +27,7 @@ main = do
   hspec $ do
     when shouldRunDoctests $
       describe "doctest" $
-        it "runs module documentation examples" doctests
+        it "should run haddock examples" doctests
     describe "regexGrammar" $ for_ regexExamples $ testGrammar False regexGrammar
     describe "semverGrammar" $ for_ semverExamples $ testCtxGrammar True semverGrammar
     describe "semverCtxGrammar" $ for_ semverExamples $ testCtxGrammar True semverCtxGrammar

--- a/test/Main.hs
+++ b/test/Main.hs
@@ -5,7 +5,9 @@ import Control.Lens.Grammar
 import Control.Monad (when)
 import Data.IORef
 import Data.List (genericLength)
+import Data.Maybe (isJust)
 import Data.Profunctor.Types (Star (..))
+import System.Environment (lookupEnv)
 import Test.DocTest
 import Test.Hspec
 
@@ -21,7 +23,11 @@ import Properties.Kleene
 
 main :: IO ()
 main = do
+  shouldRunDoctests <- isJust <$> lookupEnv "DISTRIBUTORS_RUN_DOCTESTS"
   hspec $ do
+    when shouldRunDoctests $
+      describe "doctest" $
+        it "runs module documentation examples" doctests
     describe "regexGrammar" $ for_ regexExamples $ testGrammar False regexGrammar
     describe "semverGrammar" $ for_ semverExamples $ testCtxGrammar True semverGrammar
     describe "semverCtxGrammar" $ for_ semverExamples $ testCtxGrammar True semverCtxGrammar
@@ -34,8 +40,6 @@ main = do
     describe "Parsector try rollback" tryRollbackTests
     describe "Kleene" kleeneProperties
     describe "meander" meanderProperties
-    describe "doctest" $
-      it "runs module documentation examples" doctests
 
 tryRollbackTests :: Spec
 tryRollbackTests = do

